frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -408,7 +409,7 @@ function ReaderHighlight:addToMainMenu(menu_items)
         table.insert(menu_items.highlight_options.sub_item_table, {
             text_func = function()
                 local text = v[1]
-                if v[2] == G_reader_settings:readSetting("highlight_drawing_style") then
+                if v[2] == G_reader_settings:readSetting("highlight_drawing_style", self._fallback_drawer) then
```

Comment:
This will save as `highlight_drawing_style` the value of `self._fallback_drawer` once this is merged (while  `G_reader_settings:readSetting("highlight_drawing_style") or self._fallback_drawer` would not).
Just asking (no idea) if that's really what you want.
(Other similar below.)

Reply:
Modern trend is not to save.